client/src/components/CreateQuestion.vue

```diff
@@ -0,0 +1,142 @@
+<template>
+  <div>
+    <div class="container">
+      <h2>Create question</h2>
+      <input class="input-textarea" type="text" placeholder="Question" v-model="text" />
+      <ul id="answers">
+        <li v-for="answer in answers" :key="answer.localId">
+          <input type="checkbox" :value="answer.id" v-model="correctAnswers" />
+          <input class="input-textarea" type="text" placeholder="Answer" v-model="answer.text" />
+        </li>
+      </ul>
+      <button class="signup-form-button active-theme" v-on:click.prevent="addNewAnswer">Add Answer</button>
+      <button
+        class="signup-form-button active-theme"
+        v-on:click.prevent="submitQuestion"
```

Comment:
are the prevent's here necessary? if its not a button of `submit` type or even form there shouldn't be need, also it shouldn't have any effect

Reply:
Resolved